src_dualdesc/basic_datafile.h

```diff
@@ -156,6 +156,35 @@ struct FileBool {
       throw TerminalException{1};
     }
   }
+
+  size_t getpopcnt() {
```

Comment:
What is `getpopcnt` doing? I do not see it being used anywhere in the code.

Reply:
I have program to quickly compute the progress status (orbits, orbits done, orbits left) that uses it. I can polish it some more and then push it. It is useful for the large ongoing computations.

Reply:
Should officially not have been part of this PR..

Reply:
No problem, just wanted to know. You can actually add your tools to the polyhedral_common.

Reply:
And I do not advocate for atomic PRs, we are not running a tight ship like google or similar where PRs have to be done in a very orderly way.